scripts/internal/update-account-servers-database.sh

```diff
@@ -21,6 +21,7 @@ run_verbose docker compose exec -e SERVER_IP="$SERVER_IP" \
     -e MIIVERSE_AES_KEY="$PN_MIIVERSE_API_CONFIG_AES_KEY" \
     -e WIIU_CHAT_PORT="$PN_WIIU_CHAT_AUTHENTICATION_SERVER_PORT" \
     -e SMM_PORT="$PN_SMM_AUTHENTICATION_SERVER_PORT" \
+    -e SPLATOON_PORT="$PN_SPLATOON_AUTHENTICATION_SERVER_PORT" \
```

Comment:
Caught the issue in CI: this needs to load the Splatoon dotenv files on line 10.

Reply:
Was added to load the dotenv file on the right line, shouldn't be causing CI issues now